### PR TITLE
issue 1; updating paypal and adding a java 9 dependency.

### PR DESCRIPTION
**Pull request by crkoehnen**
Please don't hit merge until we get confirmation from Bridge Factory that the donation was properly earmarked for ClojureBridge MN.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -48,6 +48,8 @@
              ]
             [lein-environ "1.1.0"]]
 
+  :jvm-opts ["--add-modules" "java.xml.bind"]
+
```
Comment (crkoehnen) [inline project.clj:52]: I'm running java 9.  This change fixes the following error:

`Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:466)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:563)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
`

File: old-static-site/resources/public/individual.html [hunk 2]
```diff
@@ -46,7 +46,9 @@ <h1><a href="/">&nbsp;ClojureBridge&nbsp;MN&nbsp;</a></h1>
           <td class="center-middle" colspan="5">
             You can donate <i>right now</i>!
             <br/>
-            Click below to donate to School Factory (the non-profit that cares for Bridge Foundry and ClojureBridgeMN)
+            Click below to donate
+            to <a href="https://bridgefoundry.org">Bridge Factory</a>.
+            (All donations are provided as unrestricted funds.)
```
Comment (crkoehnen) [inline old-static-site/resources/public/individual.html]: @tmarble , this file is in `old-static-site`.  Is that directory just for reference?

Comment (tmarble) [reply old-static-site/resources/public/individual.html]: Hmm.. I think that entire directory should be removed

Comment (crkoehnen) [reply old-static-site/resources/public/individual.html]: Going, going, gone!

**Comment by crkoehnen** [comment]
Bridge Foundry confirmed the payment went through so I'm merging.  @tmarble , it automatically deploys?

**Comment by tmarble** [comment]
@crkoehnen I realize that this is an informal repo and everyone can push to master or merge their own PR's. In this case the problematic part of setting `:jvm-opts` in **project.clj** is that it assumes all users and platforms can process those options. In this case the deployment target is running Java 8 and the automatic deployment failed:

```
-- build uberjar --
Unrecognized option: --add-modules
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

I reverted that change in 8a23268 and invite you set **JVM_OPTS** in your Java 9 environment (per https://github.com/technomancy/leiningen/blob/1a366b2c01054ffcc9a32b104818bc7d1f0024c0/doc/TUTORIAL.md#setting-jvm-options )

**Comment by crkoehnen** [comment]
Doh!

Thanks @tmarble .

**Pull request merged**